src/UNL/MediaHub/AmaraAPI.php

```diff
@@ -159,11 +159,32 @@ public function getMediaHubTextTracks($media_id, $media_url, $format = 'srt')
         if ($media_details->meta->total_count == 0) {
             return $tracks;
         }
-        
+
         foreach($media_details->objects[0]->languages as $track) {
             $tracks[$track->code] = UNL_MediaHub_Controller::$url . 'media/'.$media_id.'/'.$format.'?amara_id='.$media_details->objects[0]->id.'&lang_code='.$track->code;
         }
 
         return $tracks;
     }
+
+
+    public function getTextTracks($media_url, $format = 'vtt')
+    {
+        $media_details = $this->getMediaDetails($media_url);
+        $tracks        = array();
```

Comment:
I don't think it _ever_ helps. :)
It just causes problems when a new variable needs to be declared that is longer than the rest.